lms/envs/test.py

```diff
@@ -14,20 +14,25 @@
 # pylint: disable=wildcard-import, unused-wildcard-import
 from collections import OrderedDict
 
-from django.utils.translation import ugettext_lazy
+from __future__ import absolute_import
```

Comment:
I'm not sure why ISort put this here,  the `__futurue__` imports need to go above all the other imports.  I think this is what's causing most of the test failures.  Probbaly happened during the rebase.  You may want to just re-run isort.

Reply:
No problem, I'll have a look around.